docs/content/preview/develop/change-data-capture/using-logical-replication/monitor.md

```diff
@@ -71,6 +72,33 @@ Provides a list of all replication slots that currently exist on the database cl
 | yb_stream_id | text | UUID of the CDC stream |
 | yb_restart_commit_ht | int8 | A uint64 representation of the commit Hybrid Time corresponding to the `restart_lsn`. This can be used by the client (like YugabyteDB connector) to perform a consistent snapshot (as of the `consistent_point`) in the case when a replication slot already exists. |
 
+### pg_stat_replication
+
+Displays information about active WAL senders, providing insights into the state of replication for each connected standby or logical replication client.
+
+| Column name | Data type | Description |
+| :----- | :----- | :----- |
+| pid | integer | Process ID of WAL sender process. |
+| usesysid | oid | OID of the user logged into this WAL sender process. |
+| usename | name | Name of the user logged into this WAL sender process. |
+| application_name | text | Name of the application that is connected to this WAL sender. |
+| client_addr | inet | IP address of the client connected to this WAL sender. If this field is null, it indicates that the client is connected via a Unix socket on the server machine. |
+| client_hostname | text | Host name of the connected client, as reported by a reverse DNS lookup of client_addr. This field will only be non-null for IP connections, and only when log_hostname is enabled. |
+| client_port | integer | TCP port number that the client is using for communication with this WAL sender, or -1 if a Unix socket is used. |
+| backend_start | timestamp with time zone | Time when this process was started, i.e., when the client connected to this WAL sender. |
+| backend_xmin | xid | The oldest transaction the client is interested in. |
+| state | text | Current WAL sender state (Always `streaming`). |
+| sent_lsn | pg_lsn | Last write-ahead log location sent on this connection. |
+| write_lsn | pg_lsn | The last WAL location acknowledged by the logical replication client. |
+| flush_lsn | pg_lsn | Same as `write_lsn`. The last WAL location acknowledged by the logical replication client. |
+| replay_lsn | pg_lsn | Same as `write_lsn`. The last WAL location acknowledged by the logical replication client. |
+| write_lag | interval | The lag between when a WAL record was sent and when its acknowledgment was received. Since YugabyteDB does not differentiate between write, flush, or replay, this value is the same for all three lag metrics. |
```

Comment:
This is not correct. We display the difference between the timestamps of latest record in WAL and the last record for which an acknowledgement was received.

See the description of `cdcsdk_flush_lag` metric from here: https://docs.yugabyte.com/stable/develop/change-data-capture/using-logical-replication/monitor/#cdc-service-metrics